website/docs/r/policy_context_profile.html.markdown

```diff
@@ -0,0 +1,76 @@
+---
+layout: "nsxt"
+page_title: "NSXT: nsxt_policy_context_profile"
+sidebar_current: "docs-nsxt-resource-policy-context-profile"
+description: A resource to configure a Context Profile.
+---
+
+# nsxt_policy_context_profile
+
+This resource provides a method for the management of a Context Profile.
+ 
+## Example Usage
+
+```hcl
+resource "nsxt_policy_context_profile" "test" {
+    display_name      = "test"
+    description       = "Terraform provisioned ContextProfile"
+    domain_name {
+        description = "test-domain-name-attribute"
+        value       = ["*-myfiles.sharepoint.com"]
+    }
+    app_id {
+        description = "test-app-id-attribute"
+        value       = ["SSL"]
+        sub_attribute {
+            tls_version = ["SSL_V3"]
+        }
+    }
+}
+
+```
+
+## Argument Reference
+
+The following arguments are supported:
+Note: At least one of `app_id`, `domain_name`, or `url_category` must present.
+
+* `display_name` - (Required) Display name of the resource.
+* `description` - (Optional) Description of the resource.
+* `tag` - (Optional) A list of scope + tag pairs to associate with this resource.
+* `nsx_id` - (Optional) The NSX ID of this resource. If set, this ID will be used to create the resource.
+* `app_id` - (Optional) A block to specify app id attributes for the context profile. Only one block is allowed.
+  * `description` - (Optional) Description of the attribute.
+  * `value` - (Required) A list of string indicating values for the `app_id`. Must be a subset of valid values for `app_id` on NSX.
+  * `sub_attribute` - (Optional) A block to specify sub attribute for the `app_id`. Only one block is allowed.
```

Comment:
Can we add a note here that certain types of sub attributes are applicable to certain types of apps?

Reply:
Sure. Thanks.